### ISSUE TITLE
How to deal with oov tokens with pretrained models

### ISSUE THREAD

**Issue by skurzhanskyi** [huggingface/pytorch-transformers#1230]
## ❓ Questions & Help

Can you, please, give an advice on how to handle out of vocabulary word? Just to use `[UNK]` token or there is a way to add this token to vocabulary and thus to train embedding for it?
Also, I noticed that oov words by default return multiple tokens. In my task (sequence tagging) I would like to save token-label correspondence.

**Comment by LysandreJik** [comment]
Hello! We have a method called `add_tokens` in our tokenizers that does just that. [Here's the relevant information](https://huggingface.co/pytorch-transformers/main_classes/tokenizer.html#pytorch_transformers.PreTrainedTokenizer.add_tokens) in the documentation.

**Comment by skurzhanskyi** [comment]
Thanks a lot for your answer. That's exactly what I was looking for.
